doc/nrf/libraries/networking/nrf_cloud.rst

```diff
@@ -158,10 +158,15 @@ For example, a device that supports all the FOTA types writes the following data
    }}}}}
 
 You can initiate FOTA updates through `nRF Cloud`_ or by using the `nRF Cloud Device API`_.
-When the device receives the :c:enumerator:`NRF_CLOUD_EVT_FOTA_DONE` event, the application must perform any necessary cleanup and perform a reboot to complete the update.
+When the device receives FOTA update information from nRF Cloud, the :c:enumerator:`NRF_CLOUD_EVT_FOTA_START` event is sent to the application.
```

Comment:
no, the events are sent from (this) nrf_cloud fw library.
the FOTA update information from nRF Cloud is the signal to start.

Reply:
How about:

```
When the device receives FOTA update information from nRF Cloud, the :c:enumerator:`NRF_CLOUD_EVT_FOTA_START` event is sent to the application, and the application must now perform a reboot to complete the update.
```

Reply:
How about what Georges wrote, minus the reboot comment, since you removed that as part of your PR anyway?

```
When the device receives FOTA update information from nRF Cloud, the nRF Cloud library sends the :c:enumerator:`NRF_CLOUD_EVT_FOTA_START` event to the application.
```
Alternatively, say the same, but add:
```
to the application's registered FOTA callback.
```

Reply:
applied. 

`registered FOTA callback.` - this does not exist.  there is just one callback/event handler for all nrf_cloud lib events.

the reboot comment is still there, _after_ the DONE event is received.